ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -475,17 +475,19 @@ export function PreviewConcierge(props: any) {
             `/static-assets/images/${payload.record.modelId}`,
             xsrfArgument
           ).subscribe(
-            ({ payload: { progress } }) => {
-              const percentage = Math.floor(
-                parseInt(((progress.bytesUploaded / progress.bytesTotal) * 100).toFixed(2))
-              );
-              hostToGuest$.next({
-                type: DESKTOP_ASSET_UPLOAD_PROGRESS,
-                payload: {
-                  record: payload.record,
-                  percentage
-                }
-              });
+            ({ type, payload: { progress } }) => {
+              if (type !== 'upload-success') {
```

Comment:
Unsafe check. Check for the specific event we're waiting for (i.e. progress).
Remove the if and `.pipe(filter({type} => type === progress), pluck(payload))`